README.md

```diff
@@ -19,12 +19,19 @@ aura -A wf-recorder-git
 
 ## From Source
 =======
-### install dependences (ubuntu)
+### Install Dependencies
+
+#### Ubuntu
 ```
 sudo apt install libavutil-dev libavcodec-dev libavformat-dev libswscale-dev libpulse-dev
 ```
 
-### download & build
+#### Fedora
+```
+$ sudo yum install wayland-devel wayland-protocols-devel ffmpeg-devel
```

Comment:
yum? Should be dnf.

Reply:
```suggestion
$ sudo dnf install wayland-devel wayland-protocols-devel ffmpeg-devel
```
`yum` worked for me on Fedora 31. Your call

Reply:
yum has been deprecated since a while. I am not sure how long it will work, probably the Fedora guys will be supporting it as alias for dnf for a long time, but I see no reason to use yum.

Reply:
Oh right, wasn't aware - Thanks for the info 🙂